README.md

```diff
@@ -17,10 +24,9 @@ Then:
 $ cargo install diesel_cli --no-default-features --features postgres
 ```
 
-Once you have the requisite dependencies, you can get set up by running:
+Once you have the required dependencies, you can get get developing by running the datastore for the indexer within a container in the background. After the Postgres DB is running start whichever serves you are looking to develop on:
```

Comment:
`./start-developing.sh` sets up docker-compose and diesel.  Because of how diesel works this actually needs to be run before one can start developing, not just before they can start a server.